TSC-Charter.md

```diff
@@ -176,10 +187,71 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public.
+
+Following the completion of all votes, a public statement must be made via
+GitHub specifying the results of the vote. Contextual detail about why the
+vote was held, including a listing of the specific questions voted on must be
+included in the statement.
+
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the Node.js
 Foundation board.
 
+### Votes to Remove Members
```

Comment:
I know you worked hard on this section and it's a lot of text and I'm sure it represents a lot of thought. But I disagree with it and think it should simply be removed. Go back to the original text on the matter which was something like:

> TSC members may be removed by a standard TSC motion.

Trying to figure out a process ahead of time is bound to fail, especially if it's people like us trying to figure it out.

I get that this is trying to address recent events, but IMO the issue there isn't that the TSC didn't know how to remove someone. The how-to-remove-someone process is actually straightforward. It's that the TSC didn't know how to deal with CoC stuff.

If we could do it all over again, I might suggest that the whole thing be handled like a standard TSC motion. If someone wants another member removed, make a motion. A discussion ensues. If consensus cannot be reached, it goes to a vote. There does not need to be misconduct for someone to be removed. Someone can be removed for any number of reasons. Maybe they are inactive. Maybe they are difficult to work with, without being malicious. Maybe they consistently cause problems unintentionally. Maybe they just have bad judgment. Maybe it's a combination of things like that, any one of which wouldn't be a big deal, but taken together make them unsuitable for the TSC.

Creating artificial barriers for removal ("vote must be structured in terms of whether or not the specific issues listed constitute grounds for removal") seems counterproductive. If anything, it has been too *hard* to remove members from the TSC or CTC. As evidence, it has never happened, even in one case when someone was basically inactive for 2 years.

If we want to include additional process to try to prevent a repeat of this experience, maybe focus on referring disputes to third-party mediation (by someone trained in mediation/conflict resolution, not some other member of this or another committee or even the board).  And ask the Board to agree in advance that they're willing to pay for such things.